.gitignore

```diff
@@ -289,4 +289,8 @@ __pycache__/
 *.xsd.cs
 
 # UWP Generated files
-**/Generated Files/
\ No newline at end of file
+**/Generated Files/
+/Samples/BackgroundBlur/BackgroundBlur/Assets
```

Comment:
do we not need these new asset and png files?

Reply:
It's part of the separate background blur project that i need to delete, so this can be removed entirely.